packages/frontend/src/utils/ens.ts

```diff
@@ -22,3 +22,15 @@ export async function getEnsAvatar(ensNameOrAddress: string) {
     // noop
   }
 }
+
+export async function getEnsAddress(ensName: string) {
+  if (!ensName.endsWith('.eth')) {
+    return
+  }
+
+  try {
+    return provider.resolveName(ensName)
```

Comment:
I don't think the try/catch here is effective because it's missing an await after the return keyword